src/box/applier.cc

```diff
@@ -1375,6 +1375,7 @@ applier_signal_ack(struct applier *applier)
 		applier->ack_msg.txn_last_tm = (r == NULL ? 0 :
 						r->applier_txn_last_tm);
 		applier->ack_msg.vclock_sync = applier->last_vclock_sync;
+		applier->ack_msg.term = box_raft()->volatile_term;
```

Comment:
I don't remember a discussion. Maybe you talked to Vova. My opinion here is that adding `tm` and `replica_id` to request wouldn't be better, because:
- It would be duplication of the members. You would have `tm` and `replica_id` both in `xrow_header` and in `..._request` structs.
- Duplication might lead to slightly bigger parent objects. For example, `relay.last_recv_ack` would store `tm` field even though it is not needed here. The timestamp is decoded and used in `relay_reader_f()` right away, doesn't need a long storage.

Doesn't mean it can't be done, but it would simply produce other questions. Can do it if you want anyway.

Reply:
The reason is simpler - I did it out of habit from usage inside the instance where we rely on the volatile term. A persistent term is surely not worse functionally and is consistent with existing broadcasts. Changed it to non-volatile term now.

Reply:
Nice.